client/my-sites/upgrades/checkout-thank-you/style.scss

```diff
@@ -220,3 +220,73 @@
 		max-width: 634px;
 	}
 }
+
+.purchase-detail__google-voucher-code {
+	display: flex;
+	align-items: flex-start;
+	margin-top: 12px;
+	padding-bottom: 12px;
+	border-bottom: solid 1px #E9EFF3;
```

Comment:
We should always used named colors. see `_colors.scss`. If you can't find a matching color, we can lighten/darken as needed. cc @drw158